`prefer-export-from`: Add `ignoreUsedVariables` option (#1590)
sindresorhus/eslint-plugin-unicorn: docs/rules/prefer-export-from.md

import * as namespace from './foo.js';
export default namespace;
```

## Options

### ignoreUsedVariables

Type: `boolean`\
Default: `false`

> When pass `"ignoreUsedVariables": true`, if any variable is used more than just re-export or not exported, all variables in the import declaration will be ignored.

Reviewer: ```suggestion
When `true`, if an import is used in other places than just a re-export or is not exported, all variables in the import declaration will be ignored.
```

?

Author: Fixed.
